hw3/01-routing.js

```diff
@@ -34,18 +33,72 @@ const server = http.createServer((req, res) => {
     return result;
   };
 
-  if (req.url === '/') {
+
+  if (req.url === `/`) {
     let routeResults = getRoutes();
 
-    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.writeHead(200, { 'Content-Type': `text/html` });
     res.write(`<h1>Exercise 01</h1>`);
     res.write(`<ul> ${routeResults} </ul>`);
     res.end();
   }
 
-  // Add your code here
+
+  else if (req.url === `/welcome`) {
+    res.writeHead(200, {'Content-Type': `text/html`});
+    res.write(`<h1> Welcome Page </h1>`);
+    res.end();
+  }
+
+
+  else if (req.url === `/redirect`) {
+    res.writeHead(302, {'Location':'/redirected','Content-Type': `text/html`});
+    res.write(`<h1> Redirecting to the redirected Page </h1>`);
+    //res.write(`Redirected Mes`);
+    res.end();
+  }
+
+  else if (req.url === '/redirected'){
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.write(`<h1>Redirected</h1>`);
+    res.end();
+  }
+  else if (req.url === `/cache`) {
+    res.writeHead(200, {
+      'Content-Type': `text/html`,
+      'Cache-Control': `max-age=86400`
+    });
+    res.write(`This resource was cached`);
+    
+    res.end();
+  }
+  
+   else if(req.url === `/cookie`){
+    res.writeHead(200, { 'Cookie': 'hello = world' });
+    res.write('<h1> cookies yummmmmmm<h1>');
```

Comment:
in the hw3 assignment text/plain was required here. but you are sending html

Reply:
Fixed this issue